src/box/lua/config/instance_config.lua

```diff
@@ -1179,6 +1179,53 @@ return schema.new('instance_config', schema.record({
             end
         end,
     }),
+    flightrec = schema.record({
+        enabled = enterprise_edition(schema.scalar({
+            type = 'boolean',
+            box_cfg = 'flightrec_enabled',
+            default = true,
+        })),
+        logs_size = enterprise_edition(schema.scalar({
+            type = 'number',
+            box_cfg = 'flightrec_logs_size',
+            default = 10485760,
+        })),
+        logs_max_msg_size = enterprise_edition(schema.scalar({
+            type = 'number',
+            box_cfg = 'flightrec_logs_max_msg_size',
+            default = 4096,
+        })),
```

Comment:
If it is a size, I guess that it should be `integer`, not `number`. Please, re-check other options.

Reply:
Changed types of `logs_size`, `logs_max_msg_size`, `logs_log_level`, `requests_size`, `requests_max_req_size`, `requests_max_res_size` to `integer`.